plugins/http_plugin/include/eosio/http_plugin/http_plugin.hpp

```diff
@@ -159,30 +160,52 @@ namespace eosio {
       error_info error;
    };
 
+   inline bool isEmptyContent(std::string& body) {
```

Comment:
Should be snake case not camel case.
I don't expect an `is` method to modify the string.
Seems like this could be easily implemented with a `string_view` instead.

Reply:
I was wondering whether need to implement trim, for handling broader scenarios, just added in case that leading or trailing spaces being sent. 
If there is no need to trim here, rather like to change to const string& string_view

Reply:
Or, I can pull the trim outside of checking.

Reply:
Move trim out side of checking, so is method won't modify. pushed change